draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -707,20 +707,21 @@ using the MP_FAST_CLOSE suboption.
 ~~~~
 {: #ref-MP_FAST_CLOSE title='Format of the MP_FAST_CLOSE Suboption'}
 
-The MP_FAST_CLOSE suboption MUST be sent from an initiating host on all subflows 
-using a DCCP-Reset packet with Reset Code THIS-RESET-CODE. 
-To protect from unauthorized shutdown of a multipath DCCP connection, 
+When host A wants to abruptly close MP-DCCP connection with host B, it will send out the MP_FAST_CLOSE. The MP_FAST_CLOSE suboption MUST be sent from host A on all subflows 
+using a DCCP-Reset packet with Reset Code THIS-RESET-CODE. The requirement to send the MP_FAST_CLOSE on all subflows increases the probability that host B will receive the MP_FAST_CLOSE to take the same action. To protect from unauthorized shutdown of a MP-DCCP connection, 
 the selected Key Data of the peer host during the handshaking procedure 
 is carried by the MP_FAST_CLOSE option. 
 
-On completion of this step, the initiating host tears down all subflows 
+On completion of this step, host A will tear down all subflows 
```

Comment:
"this step" is a bit unclear. Do you mean after sending MP_FAST_CLOSE on all subflows.

Reply:
Text changed, 'this step' removed and changed to 'After sending the MP_FAST_CLOSE on all subflows, ...'